packages/tab/src/react/__stories__/index.story.tsx

```diff
@@ -1,14 +1,13 @@
-import PropTypes from 'prop-types'
-import React from 'react'
+import * as React from 'react'
 import { storiesOf } from '@storybook/react'
 
-import Tab from '../index.js'
+import Tab from '../../index.js'
```

Comment:
We're only typing the "public interface", so need to import from there

Reply:
Oh, wow, is this required to get the proper typings on `Tab` from within this story file?

Reply:
So this is a side effect of my most recent change, which collapses the two separate `index.d.ts` files into one, at the outer layer. The advantage of collapsing them is that it makes it easier for consumers to augment the interface to add custom props if they need to[0]. The other advantage is that there's fewer files to copy out into `dist`

I do recognize this as a downside of the change, because it means we wouldn't be able to import anything into a story that's not exported in the other file.

Happy to discuss whether this is worth it.

[0]: With two separate files, the consumer has to do something like this:

```typescript
import Tab from '@pluralsight/ps-design-system-tab'

declare module '@pluralsight/ps-design-system-tab/dist/cjs/react' {
    interface ListItemProps {
        name: string
    }
}
```

(You have to know the internals a bit, notice the `/dist/cjs/react` in the `declare module` augmentation. This is a known bug/design limitation in TS, that it isn't able to handle the re-export for the purposes of augmentation)

With a single file, we can just do:

```typescript
import Tab from '@pluralsight/ps-design-system-tab'

declare module '@pluralsight/ps-design-system-tab' {
    interface ListItemProps {
        name: string    }
}
```

Either way, we would need to document how to do this for consumers, so maybe the first option is acceptable?

Reply:
Hey Justin, I'm glad you're working on this.  I would not have anticipated this issue.

In my opinion, as a consumer, option 2 is by far better.  The collapsing into a single outer .d.ts file seems worth it to provide the consumer experience.

We good to merge, then?

Reply:
Yes, I'm happy with where this ended up if you are.

Next steps, I will test this out on the search team, and type one more component to make sure this pattern scales.